geometry_msgs/Vector3 pose
float32 weight